src/main/resources/static/openapi/EducationAndWorkPlanAPI.yml

```diff
@@ -476,6 +495,12 @@ components:
         - reference
         - prisonNumber
         - goals
+    GetGoalsResponse:
```

Comment:
I'm going to leave this as an array. We shouldn't be making it more complicated than it needs to be "just in case". If we're adding meta data than that isn't just getting goals anymore and should be a different endpoint and if we're adding pagination (highly unlikely) then it's a functionally breaking change anyway